packages/react/src/avatar/image/useImageLoadingStatus.ts

```diff
@@ -5,9 +5,14 @@ import { useEnhancedEffect } from '../../utils/useEnhancedEffect';
 
 export type ImageLoadingStatus = 'idle' | 'loading' | 'loaded' | 'error';
 
+interface UseImageLoadingStatusOptions {
+  referrerPolicy?: React.HTMLAttributeReferrerPolicy;
+  crossOrigin?: string | null;
```

Comment:
This could be narrowed to `React.ImgHTMLAttributes<HTMLImageElement>['crossOrigin']`